src/auth-service/models/Defaults.js

```diff
@@ -31,15 +32,33 @@ const DefaultsSchema = new mongoose.Schema({
     type: String,
     required: [true, "chartTitle is required!"],
   },
+  chartSubTitle: {
+    type: String,
+    required: [true, "chartSubTitle is required!"],
+  },
+  airqloud_id: {
+    type: ObjectId,
+  },
+  airqloud: {
+    type: String,
+  },
+  user_id: {
+    type: ObjectId,
+  },
   user: {
     type: ObjectId,
     required: [true, "user is required"],
   },
-  locations: [
+  sites: [
     {
       type: String,
     },
   ],
+  site_ids: [
+    {
+      type: ObjectId,
+    },
+  ],
   period: { type: String, required: [true, "period is required!"] },
```

Comment:
@ja-odur 
Please not that the `period` is supposed to just be a String, I am not yet very sure why we have "stringified" objects within our database records.

Reply:
I think I now remember why it is `stringified`. The field accepts a string but I needed to store the an object. Maybe the schema type can be changed to object so that I can remove the `stringification` @Baalmart

Reply:
Resolved. @ja-odur